src/http/proxy_server.rs

```diff
@@ -99,7 +99,7 @@ pub(crate) async fn start(
                 );
 
                 let mut response =
-                    if render_static {
+                    if render_static || php_port == 0 {
```

Comment:
In what case would `php_port` be zero?
And what is the reason to force static rendering in such case?

Reply:
Because PHP is not started (entry point not found). If you try send the request to PHP backend, an error is retourned.